doghouse/server/storage.go

```diff
@@ -0,0 +1,58 @@
+package server
+
+import (
+	"context"
+
+	"google.golang.org/appengine/datastore"
+)
+
+type Installation struct {
+	RepositoryID       int64
+	InstallationID     int
+	RepositoryFullName string
+}
+
+func newInstallationKey(ctx context.Context, repositoryName string) *datastore.Key {
+	kind := "Installation"
+	return datastore.NewKey(ctx, kind, repositoryName, 0, nil)
+}
+
+type CheckSuiteEvent struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type CheckSuiteEvent should have comment or be unexported